rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -647,6 +647,7 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
 	QAction* copy_name = info_menu->addAction(tr("&Copy Name"));
 	QAction* copy_serial = info_menu->addAction(tr("&Copy Serial"));
 
+	const std::string cache_base_dir = fs::get_cache_dir() + "cache/" + currGame.serial;
```

Comment:
`fs::get_cache_dir()` returns `fs::get_config_dir()` for _WIN32
https://github.com/RPCS3/rpcs3/blob/master/Utilities/File.cpp#L1435

Reply:
Also, can you change [this](https://github.com/RPCS3/rpcs3/blob/master/rpcs3/rpcs3qt/game_list_frame.cpp#L740) to use `config_base_dir`? It looks like someone forgot to do that.

Reply:
Settings remain in data, it's not forgotten.

Reply:
who would have guessed

Reply:
i mean that [`config_base_dir`](https://github.com/RPCS3/rpcs3/blob/master/rpcs3/rpcs3qt/game_list_frame.cpp#L650) already contains `fs::get_config_dir() + "data/" + currGame.serial` so why is the path constructed again [here](https://github.com/RPCS3/rpcs3/blob/master/rpcs3/rpcs3qt/game_list_frame.cpp#L740)?

Reply:
because we obviously dont want that

Reply:
that's the whole reason for this PR

Reply:
the minor change you suggest was already implemented in my other PR anyway

Reply:
which basically engulfs this PR

Reply:
I'll mark this as solved since changing line 740 is out of scope for this PR and the cache/config dir thing is intentional.